### ISSUE TITLE
Error while running

### ISSUE THREAD

**Issue by anmolduainter** [anshkumar/yolact#4]
I got error for importing deformable_conv2d from tensorflow addons, after checking docs there is no such layer supported by it.
How are you using it ?
from tensorflow_addons.layers.deformable_conv2d import DeformableConv2D

**Comment by anshkumar** [comment]
It's a custom addon. Will push it once I've finished testing.

**Comment by anmolduainter** [comment]
thanks for clarifying that point.
In this code inside scheduler:
    learning_rate = 0.5 * learning_rate_base * (1 + tf.cos(
        np.pi *
        (tf.cast(global_step, tf.float32) - warmup_steps - hold_base_rate_steps
        ) / tf.cast(total_steps - warmup_steps - hold_base_rate_steps, tf.float32)))
I think cyclic steps mod here is missing?
